nebari/initialize.py

```diff
@@ -38,17 +38,17 @@
         "authentication": None,
     },
     "default_images": {
-        "jupyterhub": f"quay.io/nebari/nebari-jupyterhub:{qhub_image_tag}",
-        "jupyterlab": f"quay.io/nebari/nebari-jupyterlab:{qhub_image_tag}",
-        "dask_worker": f"quay.io/nebari/nebari-dask-worker:{qhub_image_tag}",
+        "jupyterhub": f"quay.io/nebari/nebari-jupyterhub:{nebari_image_tag}",
+        "jupyterlab": f"quay.io/nebari/nebari-jupyterlab:{nebari_image_tag}",
+        "dask_worker": f"quay.io/nebari/nebari-dask-worker:{nebari_image_tag}",
     },
     "storage": {"conda_store": "200Gi", "shared_filesystem": "200Gi"},
     "theme": {
         "jupyterhub": {
             "hub_title": None,
             "hub_subtitle": None,
             "welcome": None,
-            "logo": "/hub/custom/images/jupyter_qhub_logo.svg",
+            "logo": "/hub/custom/images/jupyter_nebari_logo.svg",
```

Comment:
```suggestion
            "logo": "/hub/custom/images/Nebari-Logo-Horizontal-Lockup-White-text.svg",
```
as per https://github.com/nebari-dev/nebari-jupyterhub-theme/blob/641efe948594902e469a554dc4556495bb3b37ae/test_jupyterhub_config.py#L23